test/parallel/parallel.status

```diff
@@ -18,4 +18,3 @@ test-npm-install:      PASS,FLAKY
 [$system==solaris] # Also applies to SmartOS
 
 [$system==freebsd]
-
```

Comment:
Unrelated change. Please add that back in.

Reply:
It's odd because I have this last line in my editor

Reply:
This shows two lines at the end of the file (i.e. the file ends in `\n\n` - one shows up as a new line, and the other as an EOF `\n` which is required...)